gateway/api/views.py

```diff
@@ -436,9 +436,16 @@ def logs(self, request, pk=None):  # pylint: disable=invalid-name,unused-argumen
         tracer = trace.get_tracer("gateway.tracer")
         ctx = TraceContextTextMapPropagator().extract(carrier=request.headers)
         with tracer.start_as_current_span("gateway.job.logs", context=ctx):
-            job = self.get_object()
+            job = Job.objects.filter(pk=pk).first()
             logs = job.logs
-        return Response({"logs": logs})
+            author = self.request.user
+            if job.program and job.program.provider:
+                if job.program.provider.admin_group in author.groups.all():
+                    return Response({"logs": logs})
+                return Response({"logs": "No available logs"})
+            if author == job.author:
```

Comment:
I would like to know your opinion too @akihikokuroda @psschwei , because the logic is correcto but maybe here instead of an if it's easier to understand if we write `else if author == job.author:`

The logic is correct but it cost me to figure out that the second step not runs if someone enters in the first one?

Reply:
I tried but the lint doesn't like it. 
```
api/views.py:444:12: R1705: Unnecessary "elif" after "return", remove the leading "el" from "elif" (no-else-return)
```

Reply:
aaaaah, I see, thanks Aki!